xbmc/utils/URIUtils.cpp

```diff
@@ -777,6 +777,24 @@ bool URIUtils::IsDVD(const std::string& strFile)
   return false;
 }
 
+bool URIUtils::IsDVDFile(const std::string& strFile)
+{
+  std::string strFileLow = strFile;
+  StringUtils::ToLower(strFileLow);
+  if (strFileLow.find("video_ts.ifo") != std::string::npos)
+    return true;
+  return false;
+}
+
+bool URIUtils::IsBDFile(const std::string& strFile)
+{
+  std::string strFileLow = strFile;
+  StringUtils::ToLower(strFileLow);
+  if (strFileLow.find("index.bdmv") != std::string::npos)
+    return true;
+  return false;
```

Comment:
`return (fileLow.find("index.bdmv") != std::string::npos));`

Reply:
The implementation is not correct. Same problem as `IsDVDFile`.

Reply:
Done